package.json

```diff
@@ -32,29 +32,32 @@
     "verify-suite:polygon:mumbai-staging": "npx hardhat verify-suite --network mumbai --chain-id 80001 --env mumbai-staging >> logs/mumbai-staging.verify.contracts.txt",
     "verify-suite:polygon:mainnet": "npx hardhat verify-suite --network polygon --chain-id 137 --env polygon >> logs/polygon.verify.contracts.txt",
     "deploy-mocks:hardhat": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-mock-nft-auth --network hardhat",
     "deploy-mocks:local": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-mock-nft-auth --network localhost",
     "deploy-mocks:test": "npx hardhat clean && npx hardhat compile && npx hardhat deploy-mock-nft-auth --network test >> logs/test.deploy.mocks.txt",
-    "upgrade-facets:local": "npx hardhat clean && npx hardhat compile && npx hardhat run --network localhost scripts/upgrade-facets.js",
-    "upgrade-facets:test": "npx hardhat clean && npx hardhat compile && npx hardhat run --network test scripts/upgrade-facets.js  >> logs/test.upgrade.contracts.txt",
-    "upgrade-facets:ethereum:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat run --network mainnet scripts/upgrade-facets.js >> logs/mainnet.upgrade.contracts.txt",
-    "upgrade-facets:polygon:mumbai": "npx hardhat clean && npx hardhat compile && npx hardhat run --network mumbai scripts/upgrade-facets.js >> logs/mumbai.upgrade.contracts.txt",
-    "upgrade-facets:polygon:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat run --network polygon scripts/upgrade-facets.js >> logs/polygon.upgrade.contracts.txt",
-    "manage-roles:ethereum:mainnet": "npx hardhat --network mainnet run scripts/manage-roles.js >> logs/mainnet.manage.roles.txt",
-    "manage-roles:polygon:mumbai": "npx hardhat --network mumbai run scripts/manage-roles.js >> logs/mumbai.manage.roles.txt",
-    "manage-roles:polygon:mainnet": "npx hardhat --network polygon run scripts/manage-roles.js >> logs/polygon.manage.roles.txt",
-    "manage-roles:local": "npx hardhat --network localhost run scripts/manage-roles.js >> logs/localhost.manage.roles.txt",
-    "manage-roles:test": "npx hardhat --network test run scripts/manage-roles.js >> logs/test.manage.roles.txt",
+    "upgrade-facets:local": "npx hardhat clean && npx hardhat compile && npx hardhat upgrade-facets --network localhost",
+    "upgrade-facets:test": "npx hardhat clean && npx hardhat compile && npx hardhat upgrade-facets --network test >> logs/test.upgrade.contracts.txt",
+    "upgrade-facets:ethereum:mainnet": "npx hardhat clean && npx hardhat compile && npx hardhat upgrade-facets --network mainnet >> logs/mainnet.upgrade.contracts.txt",
+    "upgrade-facets:polygon:mumbai-test": "npx hardhat clean && npx hardhat compile && npx hardhat upgrade-facets --network mumbai --env test >> logs/mumbai-test.upgrade.contracts.txt",
```

Comment:
This won't be a problem for the staging upgrade. 
Script compares version in `package.json` (2.1.0) to the version in `80001-mumbai-staging.json` (2.0.0-rc.5) so it will pass without the prompt.

But still thanks for spotting this, since in some cases this could really be a problem.
For a moment I solved it by changing prompt type from stdout so stderr which is not redirected to the file and normally shows in the terminal.
Alternatively we could use `| tee` to always output everything to terminal and file at the same time.